config-overrides/manifest.overrides.js

```diff
@@ -24,7 +18,7 @@ function firefoxGeckoview(manifest) {
 /**
  * @param {typeof base} manifest
  */
-function firefoxDesktop(manifest) {
+function firefoxDesktopAndAndroid(manifest) {
```

Comment:
Just remove this one and call it "firefox"

Reply:
it's confusing because we have gecko which also is firefox

Reply:
it's not hard, it's just a `geckoViewManifest extends firefoxManifest` concept